### PR TITLE
make (most) of the project build and work on windows

### PR DESCRIPTION
**Pull request by radai-rosenblatt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ LinkedIn.
 
 ### Supported versions of Avro ###
 
-The helper module supports avro 1.4 - 1.9 inclusive. for fastserde support: 
+The helper module supports avro 1.4 - 1.10 inclusive. for fastserde support: 
 
 | Version  | Serialization | Deserialization | Fast Serialization | Fast Deserialization |
 | -------- | ------------- | --------------- | ------------------ | -------------------- |
```
Comment (FelixGV) [inline README.md:75]: The whole matrix is supported now... Do you mind deleting this?

File: avro-codegen/build.gradle [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright 2018 LinkedIn Corp.
- * Licensed under the BSD 2-Clause License (the "License"). 
+ * Licensed under the BSD 2-Clause License (the "License").
```
Comment (FelixGV) [inline avro-codegen/build.gradle:3]: What happened here? IDE going rogue on you? There's lots of these trivial changes in your PR...?

**Review by FelixGV — COMMENTED** [review]
Left minor comments. Looks ok.

File: avro-codegen/build.gradle (hunk 2 above)
Comment (radai-rosenblatt) [reply avro-codegen/build.gradle:3]: some non-printable character was giving the build issues

**Pull request merged**